deep_qa/models/text_classification/true_false_model.py

```diff
@@ -59,6 +59,7 @@ def _instance_type(self):
     @overrides
     def _set_max_lengths_from_model(self):
         self.max_sentence_length = self.model.get_input_shape_at(0)[1]
```

Comment:
This should just be a single call to the super class: `super(TrueFalseModel, self)._set_max_lengths_from_model_input(self.model.get_input_shape_at(0)[1:]`